### PR TITLE
PR to improve API calls and create uniformity

### PR DESCRIPTION
**Pull request by souvikmishra**

### CODE REVIEW

**Review by vishwajeetraj11 — APPROVED** [review]

File: src/lib/data-stores/oysterStore.ts [hunk 1]
```diff
@@ -517,7 +517,8 @@ export function initializeInventoryDataInOysterStore(oysterJobs: OysterInventory
 	oysterStore.update((value) => {
 		return {
 			...value,
-			jobsData: [...combineAndDeduplicateJobs(oysterJobs, value.jobsData)],
+			jobsData: oysterJobs,
```
Comment (vishwajeetraj11) [inline src/lib/data-stores/oysterStore.ts:520]: how is the deduplication part taken care of now?

**Review by vishwajeetraj11 — APPROVED** [review]

File: src/routes/bridge/+layout.svelte [hunk 2]
```diff
@@ -17,6 +33,34 @@
 		? $allowedChainsStore.includes($chainStore.chainId)
 		: true;
 
+	// TODO:  update the allowance function to fetch from correct subgraph
```
Comment (nayanmarlin) [inline src/routes/bridge/+layout.svelte:36]: @souvikmishra - Can you follow our decided method for TODO?

Eg. @Souvik - Todo: <<<< ---Text you want to mention --->>>>

**Review by nayanmarlin — DISMISSED** [review]

File: src/lib/data-stores/oysterStore.ts (hunk 1 above)
Comment (souvikmishra) [reply src/lib/data-stores/oysterStore.ts:520]: its not being taken care of as of now, had to remove this part as it was showing up jobs of different chains on the same page if we switched chains

File: src/routes/bridge/+layout.svelte (hunk 2 above)
Comment (souvikmishra) [reply src/routes/bridge/+layout.svelte:36]: sure

**Review by nayanmarlin — APPROVED** [review]

**Pull request merged**